eng/tsp-core/pipelines/jobs/e2e.yml

```diff
@@ -19,6 +19,9 @@ jobs:
       - script: pnpm run test:e2e
         displayName: E2E Tests
 
+      - script: pnpm run generate-scenarios-summary
```

Comment:
I mean just add it as part of the `test:e2e` or create a `regen-docs` script in the docs this is otherwise another command people will need to know to run for the workspace